bin/pytest-utils/generate-report

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
You know this will cause problems for us, right? :) Does this actually require python3 for anything? I assume we can just trivially make it compatible between both 2 and 3. Otherwise, you're going to have to like, install python3 on everyone's computers and also install it in Travis.

Reply:
IIRC Travis actually has `python3`, the `version: 2.7` directive just means venv for 2.7 is activated.

`/usr/bin/env python3` will invoke Python 3 regardless of whether you are in a virtualenv or not. Try `which python python3` in a 2.7 venv:

```sh
(sentry) [josh@akashi sentry (tests/new-utils ✗)]$ which python python3
/Users/josh/.virtualenvs/sentry/bin/python
/usr/local/bin/python3
```

Reply:
So no action is required on anyone's behalf 😀

Reply:
But that still requires you to have python3 installed to run it.

That is good to know that Travis has python3 though. For example, on my computer right now, I don't have python3.

```
$ /usr/bin/env python3
env: python3: No such file or directory
```

Reply:
Whoops, I forgot. Probably did `brew install python` instinctively when I first installed homebrew on the work laptop.

Reply:
Okay, python 2 compat has been added.